docs/how-to/deploy_sdcore_production.md

```diff
@@ -0,0 +1,328 @@
+# Deploy a Single Node Production SD-Core
+
+This guide covers how to install a SD-Core 5G core network on a single node for production.
+
```

Comment:
We should probably link to the reference somewhere here. 

```
see also: production deployment reference
```

I added the same comment on the reference page so that it's easy to navigate between the 2.

Reply:
I added the link.